comp/core/secrets/secretsimpl/secrets_mock.go

```diff
@@ -32,20 +54,20 @@ func (m *MockSecretResolver) SetFetchHookFunc(f func([]string) (map[string]strin
 	m.fetchHookFunc = f
 }
 
+// NewMock returns a MockSecretResolver
+func NewMock(testDeps testDeps) MockProvides {
```

Comment:
Do you mind leaving the function private as it was? I rather avoid people using this function directly instead of `MockModule`

Reply:
fixed in https://github.com/DataDog/datadog-agent/pull/26675/commits/ff640b5a7496f8994d7598fbb545e3c03901623b